src/ui/components/PartialAccordionDescription.tsx

```diff
@@ -12,13 +12,14 @@ import { Dd } from 'ui/web/list/Dd'
 
 interface Props {
   description?: string | undefined
+  credit?: string | null | undefined
```

Comment:
`?` et `| undefined` c'est redondant

pour quelles raisons il y a `null` ?

pourquoi ne peut-on pas avoir ?
```suggestion
  credit?: string
```

Reply:
J'ai une erreur sans utiliser les 3 (peut être j'ai loupé quelque chose au niveau de l'interface mais j'ai l'impression d'avoir fait comme pour le reste)  : 

> Impossible d'assigner le type 'string | null | undefined' au type 'string | undefined'.
  Impossible d'assigner le type 'null' au type 'string | undefined'

Reply:
ah ! je comprends d'où vient le `null`

https://github.com/pass-culture/pass-culture-app-native/pull/2793/files#diff-166893c5f8636d45cc172736661eef1f26aa4264b475e7ca7649de26746ce907R2127

je pense que tu peux enlever `| undefined`

```suggestion
  credit?: string | null
```

Reply:
Bien vu :)